resources/zib-Patient.xml

```diff
@@ -0,0 +1,468 @@
+      </type>
+      <mapping>
+        <identity value="zib-patient-v3.2-2020EN" />
+        <map value="NL-CM:0.1.4" />
+        <comment value="AddressInformation" />
+      </mapping>
+    </element>
+    <element id="Patient.maritalStatus">
+      <path value="Patient.maritalStatus" />
+      <short value="MaritalStatus" />
+      <definition value="A person’s marital status according to the terms and definition in the Dutch civil code." />
+      <alias value="BurgerlijkeStaat" />
+      <binding>
+        <strength value="required" />
+        <description value="BurgerlijkeStaatCodelijst" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.7.9.1--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-maritalstatus-v3.1-2020EN" />
+        <map value="NL-CM:7.9.2" />
+        <comment value="MaritalStatus" />
+      </mapping>
+    </element>
+    <element id="Patient.multipleBirth[x]">
+      <path value="Patient.multipleBirth[x]" />
+      <slicing>
+        <discriminator>
+          <type value="type" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Patient.multipleBirth[x]:multipleBirthBoolean">
+      <path value="Patient.multipleBirth[x]" />
+      <sliceName value="multipleBirthBoolean" />
+      <short value="MultipleBirthIndicator" />
+      <definition value="An indication stating whether the patient is of a multiple birth." />
+      <alias value="MeerlingIndicator" />
+      <type>
+        <code value="boolean" />
+      </type>
+      <mapping>
+        <identity value="zib-patient-v3.2-2020EN" />
+        <map value="NL-CM:0.1.31" />
+        <comment value="MultipleBirthIndicator" />
+      </mapping>
+    </element>
+    <element id="Patient.communication">
+      <path value="Patient.communication" />
+      <short value="LanguageProficiency" />
+      <definition value="Root concept of the LanguageProficiency information model. This concept contains all data elements of the LanguageProficiency information model." />
+      <alias value="Taalvaardigheid" />
+      <mapping>
+        <identity value="zib-languageproficiency-v3.2-2020EN" />
+        <map value="NL-CM:7.12.1" />
+        <comment value="LanguageProficiency" />
+      </mapping>
+    </element>
+    <element id="Patient.communication.extension">
+      <path value="Patient.communication.extension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Patient.communication.extension:languageControl">
+      <path value="Patient.communication.extension" />
+      <sliceName value="languageControl" />
+      <short value="LanguageControlListening-LanguageControlSpeaking-LanguageControlReading" />
```

Comment:
Is this the way? using "-" Aanvullen in de Profiling Guideline?

Reply:
Ik moet dit nog wel eerst even zien in de rendering of ik het helemaal snap .

Reply:
Beetje hiaat in de Guidelines inderdaad. Ik doe het ook al in AddressInformation

Reply:
Dit is ondertussen aangevuld, moet worden ' /  '

Reply:
Gefixed